SCENE-SCRAPABLE.md

```diff
@@ -52,6 +53,9 @@ asiantgirl.com|GroobyNetwork-Partial.yml|Trans
 assholefever.com|GammaEntertainment.yml|
 assmeat.com|Hustler.yml|
 assteenmouth.com|Teencoreclub.yml|
+atkexotics.com|ATKExotics.yml|
+atkhairy.com|ATKHairy.yml|
+atkpremium.com|ATKPremium.yml|
```

Comment:
With the other changes this can be changed to:

`atkpremium.com| ATKHairy.yml|`